c_src/brotli_nif.c

```diff
@@ -215,60 +216,60 @@ static ERL_NIF_TERM brotli_encoder_compress_stream(ErlNifEnv *env, int argc,
     return BADARG;
   }
 
-  size_t available_out = 0;
+
   size_t available_in = input.size;
   const uint8_t *next_in = input.data;
+  size_t available_out = 0;
+  uint8_t *next_out = NULL;
```

Comment:
Maybe remove them completely since they are not used?

Reply:
I think it will crash if the address is NULL.